girder/taskflow/server/taskflows.py

```diff
@@ -208,7 +213,8 @@ def terminate(self, taskflow, params):
         taskflow_instance.terminate()
 
     @access.user
-    @loadmodel(model='taskflow', plugin='taskflow', level=AccessType.ADMIN)
+    @loadmodel(model='taskflow', plugin='taskflow', level=AccessType.ADMIN,
+               requiredFlags='execute')
```

Comment:
@TristanWright That was my concern, I would have thought that requiredFlags would me that any user would need 'execute' to hit the endpoint. However, I think because you removed the registerAccessFlag call this parameter no longer does anything.